### PR TITLE
Improve Windows Format

### PR DESCRIPTION
**Pull request by lvoloshyn-sekoia**

### CODE REVIEW

File: Windows/windows/ingest/parser.yml [hunk 1]
```diff
@@ -1068,3 +1112,13 @@ stages:
       - set:
           source.address: "{{json.event.SourceHostname}}"
         filter: "{{json.event.SourceHostname != null and json.event.SourceHostname != '-'}}"
+
+      - set:
+          source.address: "{{json_event.message.SourceAddr}}"
+        filter: "{{json.event.SourceAddr != null and json.event.SourceAddr != '-'}}"
```
Comment (PierrickV) [inline Windows/windows/ingest/parser.yml:1118]:
```suggestion
```
You can remove this filter as placeholder values will be removed by:
- https://github.com/SEKOIA-IO/intake-formats/pull/815

File: Windows/windows/ingest/parser.yml [hunk 2]
```diff
@@ -358,6 +359,49 @@ stages:
...
+          action.properties.NASPort: "{{json_event.message.NASPort}}"
+          action.properties.ClientName: "{{json_event.message.ClientName}}"
+          action.properties.ClientIPAddress: "{{json_event.message.ClientIPAddress}}"
+          action.properties.ProxyPolicyName: "{{json_event.message.ProxyPolicyName}}"
+          action.properties.NetworkPolicyName: "{{json_event.message.NetworkPolicyName}}"
+          action.properties.AuthenticationProvider: "{{json_event.message.AuthenticationProvider}}"
+          action.properties.AuthenticationServer: "{{json_event.message.AuthenticationServer}}"
+          action.properties.AuthenticationType: "{{json_event.message.AuthenticationType}}"
+          action.properties.EAPType: "{{json_event.message.EAPType}}"
+
```
Comment (PierrickV) [inline Windows/windows/ingest/parser.yml:404]: Could you move action.properties fields to the dedicated stage ?

https://github.com/SEKOIA-IO/intake-formats/blob/7df70e5ce335285ece986e0639fbdf4984dc7d51/Windows/windows/ingest/parser.yml#L295

File: Windows/windows/ingest/parser.yml [hunk 3]
```diff
@@ -358,6 +359,49 @@ stages:
...
         filter: "{{json.event.SourceName == 'AD FS Auditing'}}"
 
+      - set:
+          action.properties.AdditionalActionsID: "{{ json.event.get('Additional Actions ID') }}"
+          action.properties.AdditionalActionsString: "{{ json.event.get('Additional Actions String') }}"
+
+      - set:
+          action.properties.CertIssuerName: "{{json_event.message.CertIssuerName}}"
+          action.properties.CertSerialNumber: "{{json_event.message.CertSerialNumber}}"
+          action.properties.CertThumbprint: "{{json_event.message.CertThumbprint}}"
```
Comment (PierrickV) [inline Windows/windows/ingest/parser.yml:369]: If the destination field as the same name that the source field, it should be automatically handled by 

https://github.com/SEKOIA-IO/intake-formats/blob/7df70e5ce335285ece986e0639fbdf4984dc7d51/Windows/windows/ingest/parser.yml#L298

you only need to declare the field in fields.yml

**Review by PierrickV — CHANGES_REQUESTED** [review]

**Comment by PierrickV** [comment]
Thank you for this work :pray: :partying_face: 

See my comments for suggestion of improvements

File: Windows/windows/ingest/parser.yml (hunk 1 above)
Comment (lvoloshyn-sekoia) [reply Windows/windows/ingest/parser.yml:1118]: Good to know, I'll remove those

File: Windows/windows/ingest/parser.yml (hunk 2 above)
Comment (lvoloshyn-sekoia) [reply Windows/windows/ingest/parser.yml:404]: Thanks, good to know

Comment (lvoloshyn-sekoia) [reply Windows/windows/ingest/parser.yml:404]: They're actually there

**Review by PierrickV — APPROVED** [review]

**Pull request merged**